### ISSUE TITLE
Can't Select any folder to start the app

### ISSUE THREAD

**Issue by Gekonatic** [mihonapp/mihon#934]
### Steps to reproduce

Just open for the first time

### Expected behavior

I Open it, it asks to select a folder before starting 

### Actual behavior

When i Select a Folder I get sent back to the start when it asks to select the color scheme

### Mihon version

0.16.5

### Android version

Android 8.1.0

### Device

Samsung J2Core

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have gone through the [FAQ](https://mihon.app/docs/faq/general) and [troubleshooting guide](https://mihon.app/docs/guides/troubleshooting/).
- [X] I have updated the app to version **[0.16.5](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by Gekonatic** [comment]
Solved
